tests/noto_fonts_for_android_test.py

```diff
@@ -1,4 +1,5 @@
 import collections
+from fontbakery.utils import get_name_entry_strings
```

Comment:
if you are using a new dependency you need to add it to requirements.txt

Reply:
(also it seems like overkill to depend on bakery just for this, though if you want more from it that makes sense)

Reply:
modified requirements.txt ... but I haven't thought about just simplifying the interface (yes, it's heavyweight)